### ISSUE TITLE
apiclient should have version number

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Dragon-Born/google-api-python-client#119]
```
apiclient.VERSION should exist and be set to the release version of the 
library. Same for oauth2client.VERSION.

```

Original issue reported on code.google.com by `jcgregorio@google.com` on 16 Mar 2012 at 7:52

**Comment by GoogleCodeExporter** [comment]
```
Out for review http://codereview.appspot.com/5846050/
```

Original comment by `jcgregorio@google.com` on 16 Mar 2012 at 8:54

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision fc1578d85918.
```

Original comment by `jcgregorio@google.com` on 21 Mar 2012 at 7:21
- Changed state: **Fixed**
